felix/rules/endpoints.go

```diff
@@ -744,7 +749,13 @@ func (g *PolicyGroup) ChainName() string {
 }
 
 func (g *PolicyGroup) ShouldBeInlined() bool {
-	return len(g.PolicyNames) <= 1
+	var count int
+	for _, name := range g.PolicyNames {
```

Comment:
No need to finish counting all of the SNPs, return false if the count has exceeded the max value.

```
    count := 0
    for _, name := range g.PolicyNames {
        if !model.PolicyIsStaged(name) {
            count++
            if count > 1 {
                return false
            }
        }
    }
    return true
 ```

Reply:
Thanks. Nice hint. Will fix it.

Reply:
Fixed.